app/controllers/application_controller.rb

```diff
@@ -169,11 +169,11 @@ def instrument_with_timer
   # given cache_key and fills out custom response headers.
   # If the attempt results in a cache miss, then the response is generated normally and
   # will be stored in the cache.
-  def fetch_from_or_store_in_cache(skip_cache, cache_key, httpdate, event_name)
+  def fetch_from_or_store_in_cache(skip_cache, cache_key, httpdate, _event_name)
```

Comment:
If you're not longer using `event_name`, I think it would be better to just eliminate the parameter. According to my editor this function is only called once, in `samples_controller.rb` so it should be a simple modification.

Reply:
Oops, good catch. That's leftover from the previous call to the other logging function

Reply:
Don't forget to change the call's arguments over in the samples controller as well :)
https://github.com/chanzuckerberg/idseq-web/blob/82c24cb7ff91a9311201a4b44f761c434cc64c4a/app/controllers/samples_controller.rb#L773